### PR TITLE
Issue 649 Create only one request invitation

### PR DESCRIPTION
**Pull request by rjourde**

### CODE REVIEW

File: helpers/messages.go [hunk 1]
```diff
@@ -59,7 +59,8 @@ const (
 	ErrorCodeTeamRequestNotFound      = "Request not found"
 	ErrorCodeTeamMemberNotFound       = "Member not found"
 	ErrorCodeTeamAdminCannotLeave     = "Team administrator cannot leave the team"
-	ErrorCodePrivateTeamJoinForbiden  = "Private Team cannot be joined without consent. Please request an invitation."
+	ErrorCodePrivateTeamJoinForbiden  = "Private Team cannot be joined without consent. Please request an invitation"
+	ErrorCodeTeamRequestAlreadySent   = "Sorry, you already requested an invitation"
```
Comment (santiaago) [inline helpers/messages.go]: All team errors start with team I think.
could you rename
`ErrorCodePrivateTeamJoinForbiden`  to `ErrorCodeTeamPrivateJoinForbiden`

Comment (rjourde) [inline helpers/messages.go]: Done!

**Comment by santiaago** [comment]
you are not using that code?

**Comment by rjourde** [comment]
Sorry I missed that. Done!

**Pull request merged**